### DISCUSSION TITLE
Failed to send email: com.sun.mail.smtp.SMTPSendFailedException:  STARTTLS is required to send mail

### DISCUSSION THREAD

**Discussion by vijaykumarv7** [keycloak/keycloak#13915] [Archive]
Have recently upgrade the keycloak app version from 12.0.4 (wildfly) to 18.0.2(quarkus).

 During the Keycloak function testing, Observersed that In realm under the Email tab when tried to do Test connection to send an email with configured SMTP(outlook) server.  it showing **Error! An unexpected server error has occurred** and when checked the Pod logs it showed the below error post this error. it didn't produce any error at all. Have enabled the STARTTLS SMTP configuration.

**Failed to send email: com.sun.mail.smtp.SMTPSendFailedException:  STARTTLS is required to send mail [DBBPR09CA0030.eurprd09.prod.outlook.com]**

Need urgent support.

**Comment by cgeorgilakis** [comment]
Strange. For us in the same version email with starttls is working.
